ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -880,13 +906,19 @@ def set_waypoints_velocity(self):
                     recalc = self.produce_slowdown(self.final_waypoints_start_ptr,
                                                self.lookahead_wps,
```

Comment:
Do we need to check for the condition where the red light activates when driving at high speed and dist_to_tl is within creep_zone, but shorter than min_stop_distance to prevent trying to slowdown too fast?

Reply:
This whole area needs to be clearer.  I will add the warning here, but want to change to something like this:
```
if state is stopped or creeping in creeping zone:
    creep
if state is slowdown with target matching next_tl:
    slowdown
if state is speedup or maintainspeed and next_tl is between stopping_distance and min_stopping_distance:
    start slowdown
if state is speedup or maintainspeed and next_tl is closer than min_stopping_distance:
    panic
'''